graphql_api/tests/test_test_analytics.py

```diff
@@ -0,0 +1,226 @@
+from datetime import UTC, date, datetime, timedelta
+
+from django.test import TransactionTestCase
+from freezegun import freeze_time
+
+from codecov_auth.tests.factories import OwnerFactory
+from core.tests.factories import RepositoryFactory
+from reports.tests.factories import DailyTestRollupFactory, TestFactory
+
+from .helper import GraphQLTestHelper
+
+
+@freeze_time(datetime.now().isoformat())
```

Comment:
These are copied over from those marked above for deletion in `test_repository.py`